Exiled.Events/Patches/Events/Server/RoundEnd.cs

```diff
@@ -46,18 +45,40 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             const string LeadingTeam = "<leadingTeam>5__9";
             const string NewList = "<newList>5__3";
 
+            int offset = -1;
+            int index = newInstructions.FindIndex(x => x.Calls(Method(typeof(PlayerRolesUtils), nameof(PlayerRolesUtils.GetTeam), new Type[] { typeof(ReferenceHub), }))) + offset;
+
+            Label labelcontinu = generator.DefineLabel();
```

Comment:
```suggestion
            Label continue = generator.DefineLabel();
```

Reply:
you can't name a variable continue or return